llvm/include/llvm/CodeGen/ISDOpcodes.h

```diff
@@ -814,6 +814,13 @@ enum NodeType {
 
   /// TRUNCATE - Completely drop the high bits.
   TRUNCATE,
+  /// TRUNCATE_[SU]SAT - Truncate for saturated operand
+  TRUNCATE_SSAT_S, // saturate signed input to signed result -
+                   // truncate(smin(smax(x)))
```

Comment:
Could you please spell out the parameters for my eyes? [s|u][min|max] have two parameters.